packages/ui/src/ui/pages/query-tracker/QueryEditor/QueryEditor.tsx

```diff
@@ -117,16 +118,18 @@ const QueryEditorView = React.memo(function QueryEditorView({
         },
         [dispatch, onStartQuery],
     );
+    const value = useMemo(() => text || '', [text]);
+    const editorModel = useEditorModel({value, language: engine, onChange: upadteQueryText});
     return (
         <div className={b('query')}>
-            <MonacoEditor
-                editorRef={editorRef}
-                value={text || ''}
-                language={'yql'}
-                className={b('editor')}
-                onChange={upadteQueryText}
-                monacoConfig={monacoConfig}
-            />
+            <FlexSplitPane view={FlexSplitPane.HORIZONTAL} className={b('editor')}>
```

Comment:
The component should not use FlexSplitPane, we use `withSplit` where it is required